web/vitest.setup.ts

```diff
@@ -51,20 +35,12 @@ beforeEach(() => {
     },
   });
   window.location.href = url;
-  window.REPLICATED = {
-    WEB_BUILD_VERSION: import.meta.env.VITE_BUILD_VERSION,
-    API_V1_ENDPOINT: `${import.meta.env.VITE_MARKET_API_V1_ENDPOINT}/v1`,
-    API_V3_ENDPOINT: `${import.meta.env.VITE_MARKET_API_V3_ENDPOINT}/v3`,
-    REPLICATED_APP_ENDPOINT: import.meta.env.VITE_REPLICATED_APP_ENDPOINT,
-    KURL_ENDPOINT: import.meta.env.VITE_KURL_ENDPOINT,
-    ENVIRONMENT: import.meta.env.VITE_ENVIRONMENT,
-    REGISTRY_ENDPOINT: import.meta.env.VITE_REGISTRY_ENDPOINT,
-    PROXY_ENDPOINT: import.meta.env.VITE_PROXY_ENDPOINT,
-    PUSH_ENDPOINT: import.meta.env.VITE_PUSH_ENDPOINT,
-  };
 });
 
 afterEach(async () => {
+  // Clean up DOM
+  cleanup();
```

Comment:
We need to run DOM cleanup between test executions without the `global: true` for some reason 🤷.